packages/graalvm/src/main/kotlin/elide/runtime/feature/engine/NativeTransportFeature.kt

```diff
@@ -12,15 +12,32 @@
  */
 package elide.runtime.feature.engine
 
+import org.graalvm.nativeimage.Platform
 import org.graalvm.nativeimage.hosted.Feature.BeforeAnalysisAccess
 import org.graalvm.nativeimage.hosted.Feature.IsInConfigurationAccess
+import java.nio.file.Path
 import elide.annotations.internal.VMFeature
+import elide.runtime.feature.NativeLibraryFeature.NativeLibInfo
 
 /** Registers native transport libraries for static JNI. */
 @VMFeature internal class NativeTransportFeature : AbstractStaticNativeLibraryFeature() {
   companion object {
-    const val enabled = false
+    const val ENABLED = true
+
+    // Whether to enable V2 transport libraries.
+    private val enableV2Transport = System.getProperty("elide.nativeTransport.v2") == "true"
```

Comment:
flag at build time for switching between Netty's native built-ins and our own transports